SpatialGDK/Source/SpatialGDKServices/Private/Interop/Connection/EditorWorkerController.cpp

```diff
@@ -78,9 +81,16 @@ struct EditorWorkerController
 	{
 		if (WorkerIdx < ReplaceProcesses.Num())
 		{
+			// Record a start time for timeout.
+			FDateTime BlockingStartTime =  FDateTime::Now();
```

Comment:
This timer should be started when the command is issued, not at this point.

Reply:
Not sure I follow